app/src/main/java/memphis/myapplication/data/BackgroundJobs.java

```diff
@@ -238,6 +222,7 @@ public void setup_security(Context applicationContext) {
         Globals.setFace(face);
         Globals.setMemoryCache(new MemoryCache(face, applicationContext));
         sharedPrefsManager.generateKey();
+        Globals.setUsername(sharedPrefsManager.getUsername());
```

Comment:
Why is username set here and in LoginFragment?

Reply:
Leftover from testing. Will remove.